core/templates/dev/head/App.ts

```diff
@@ -24,7 +24,8 @@ var oppia = angular.module(
     'angularAudioRecorder', 'dndLists', 'headroom', 'infinite-scroll',
     'ngAnimate', 'ngAudio', 'ngCookies', 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
     'ngResource', 'ngSanitize', 'ngTouch', 'pascalprecht.translate', 'toastr',
-    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate'
+    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate',
+    'CollectionEditorModule'
```

Comment:
AFAIK it should be this
 ``` CollectionEditorModule ``` - > ``` collectionEditorModule ``` , correct if I am wrong
Thanks!

Reply:
Hi Keshav! I am not sure either, Ill ask Ankita and Yash and rename them. about the folders, They dont affect the tests ill first get this PR to pass tests and then change the folder names. I forgot to change the names, thanks for informing! 
And for the tests, I removed node modules folder and I didnt have lan in hostel so I need to library to download large files, So i thought of directly running tests here and checking. Downloaded it now, I am fixing the tests. 
Thanks!

Reply:
Thanks @amey-kudari for informing.